packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedCallAdapter.ts

```diff
@@ -3,6 +3,8 @@
 
 import { CallWithChatAdapter } from './CallWithChatAdapter';
 import { CallAdapter, CallAdapterState } from '../../CallComposite';
+/* @conditional-compile-remove(video-background-effects) */
+import { VideoBackgroundImage } from '../../CallComposite';
```

Comment:
Should we not be getting a type like this from the Components package? are statefulClient users unable to use this type?

Reply:
this is the type of background image is only passed through composites while either initializing `adapter` or by using `updateBackgroundPickerImages`. Otherwise for components more generic form of effects item can be used to add images to picker `_VideoEffectsItem`